packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -30,6 +63,81 @@ class SingleImageViewerContainer extends React.Component {
     })
   }
 
+  dragMove (event, difference) {
+    this.setState(prevState => {
+      const { viewBox } = Object.assign({}, prevState)
+      viewBox.x -= difference.x / 1.5
+      viewBox.y -= difference.y / 1.5
+      return { viewBox }
+    })
+  }
+
+  onPan (dx, dy) {
```

Comment:
I'm making a note that the "pan view" and "zoom" actions function by modifying the `<svg>`'s `viewBox` attribute, instead of the `transform` attribute. This is fine in the context of getting this PR to work, but not standard with, say, the ["rotate view" PR](https://github.com/zooniverse/front-end-monorepo/pull/1388), which uses `transforms`.

Let me get to my full review, and I'll make a note whether it's worth discussing these different approaches.

One pertinent question though is, where did the `10` come from in `viewBox.x += dx * 10` ?

Reply:
The `10` was an arbitrary choice. We can tune it against real subjects for live projects.